typescript-functions-practice/get-last-element.ts

```diff
@@ -0,0 +1,5 @@
+/* exported getLastElement */
+function getLastElement(array: any): any {
```

Comment:
Even though you use `any` here, it should still be specified that it is an `array` of `any`